packages/fast-form-generator-react/app/index.tsx

```diff
@@ -81,7 +81,7 @@ export default class App extends React.Component<{}, IAppState> {
             <DesignSystemProvider designSystem={designSystemDefaults}>
                 <div>
                     <div
-                        style={{width: "300px", height: "100vh", background: "rgb(244, 245, 246)", float: "left"}}
+                        style={{width: "300px", minHeight: "100vh", padding: "0 8px", background: "rgb(244, 245, 246)", float: "left"}}
```

Comment:
Comment: I don't think we want this as the min-height. I think we'll want a pixel value. At a certain point, we should require a scroll or the view is going be totally broken.

Reply:
This is just for the test site (fixes background not extending the entire way) - I think it's fine here.